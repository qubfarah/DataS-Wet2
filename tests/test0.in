add_team 1
add_team 2
add_jockey 1 1
add_jockey 2 2
update_match 1 2
get_team_record 1
get_jockey_record 2
merge_teams 1 2
add_team 3
add_jockey 3 3
update_match 2 3
get_team_record 3
get_jockey_record 2
unite_by_record 1
get_team_record 1