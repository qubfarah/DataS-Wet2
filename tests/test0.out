add_team: SUCCESS
add_team: SUCCESS
add_jockey: SUCCESS
add_jockey: SUCCESS
update_match: SUCCESS
get_team_record: SUCCESS, 1
get_jockey_record: SUCCESS, -1
merge_teams: SUCCESS
add_team: SUCCESS
add_jockey: SUCCESS
update_match: SUCCESS
get_team_record: SUCCESS, -1
get_jockey_record: SUCCESS, 0
unite_by_record: SUCCESS
get_team_record: SUCCESS, 0
